src/main/java/com/code_intelligence/jazzer/mutation/mutator/lang/ByteArrayMutatorFactory.java

```diff
@@ -80,16 +95,31 @@ public byte[] detach(byte[] value) {
     public byte[] init(PseudoRandom prng) {
       // TODO: Improve the way the upper bound is determined, e.g. grow it over time and/or add
       //  support for the WithSize annotation.
-      byte[] bytes = new byte[prng.indexIn(1000)];
+      int len = prng.closedRange(minLength, maxLength);
+      byte[] bytes = new byte[len];
       prng.bytes(bytes);
       return bytes;
     }
 
     @Override
     public byte[] mutate(byte[] value, PseudoRandom prng) {
+      if (value.length == 0) {
+        return init(prng);
```

Comment:
Could you explain why this special case is needed?

Reply:
Funnily enough, I realized earlier today that it probably isn't needed because libfuzzer's mutate can probably handle an empty value array but didn't think to remove it. I can to make the implementation simpler if you'd prefer.

Reply:
In that case I would prefer to remove it, otherwise we will keep wondering about why it is there.